packages/file_selector/file_selector/pubspec.yaml

```diff
@@ -0,0 +1,21 @@
+name: file_selector
+description: Flutter plugin for opening and saving files.
+homepage: https://github.com/flutter/plugins/tree/master/packages/file_selector/file_selector
+version: 1.0.0
```

Comment:
Let's not make this 1.0.0 until it has gained some more users and stability. 1.0.0 is being used to mark plugins as "production ready" (I'm OK with... 0.7.0 :P)

Reply:
PTAL